templates/repo/branch/list.tmpl

```diff
@@ -75,7 +75,11 @@
 									</td>
 									<td class="two wide right aligned">
 										{{if not .LatestPullRequest}}
-											{{if and (not .IsDeleted) $.AllowsPulls (gt .CommitsAhead 0)}}
+											{{if .IsIncluded}}
+												<a class="ui poping up orange small label" data-content="{{$.i18n.Tr "repo.branch.included_desc"}}" data-variation="tiny inverted" data-position="top right">
```

Comment:
I don't understand why this needs data attributes. Can't it be resolved with HTML text just like the `Merged` button below? Is it because of the orange color?

The `Merged` button:
```
<a href="{{$.RepoLink}}/pulls/{{.LatestPullRequest.Issue.Index}}" class="ui purple small label"><i class="octicon octicon-git-pull-request"></i> {{$.i18n.Tr "repo.pulls.merged"}}</a>
```

Reply:
The data atribute shows the description (as seen in screnshot) wich tell the user what this mean

Reply:
We can also add popups to mergelable ... but the meaning of merge is wide known in the Git World